exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -129,18 +128,11 @@ private void AskQuestion()
             }
         }
 
-
         private string CurrentCategory()
         {
-            if (places[currentPlayer] == 0) return "Pop";
-            if (places[currentPlayer] == 4) return "Pop";
-            if (places[currentPlayer] == 8) return "Pop";
-            if (places[currentPlayer] == 1) return "Science";
-            if (places[currentPlayer] == 5) return "Science";
-            if (places[currentPlayer] == 9) return "Science";
-            if (places[currentPlayer] == 2) return "Sports";
-            if (places[currentPlayer] == 6) return "Sports";
-            if (places[currentPlayer] == 10) return "Sports";
+            if (popArray.Contains(playersPosition[currentPlayer])) return "Pop";
+            if (scienceArray.Contains(playersPosition[currentPlayer])) return "Science";
+            if (SportsArray.Contains(playersPosition[currentPlayer])) return "Sports";
```

Comment:
This is good. Maybe the arrays need clearer names? SportsQuestionsBoardPositions() ?